libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -72,19 +77,30 @@ AP_GPS_SBF::read(void)
     }
 
     if (gps._auto_config != AP_GPS::GPS_AUTO_CONFIG_DISABLE) {
-        if (_init_blob_index < ARRAY_SIZE(_initialisation_blob)) {
+        if (_init_blob_index < (ARRAY_SIZE(_initialisation_blob))) {
```

Comment:
Extra prens aren't really necessary here.

Reply:
Was a legacy of the index adjustment game, removed